test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -4241,9 +4241,15 @@ TEST_P(BalancerUpdateTest, UpdateBalancersButKeepUsingOriginalBalancer) {
   EXPECT_GT(balancers_[0]->ads_service()->eds_response_state().state,
             AdsServiceImpl::ResponseState::NOT_SENT);
   EXPECT_EQ(balancers_[1]->ads_service()->eds_response_state().state,
-            AdsServiceImpl::ResponseState::NOT_SENT);
+            AdsServiceImpl::ResponseState::NOT_SENT)
+      << "[Error Message:"
```

Comment:
No need for the square braces in the message.  Let's eliminate those, so that each of these statements is one line shorter.